ktor-server/ktor-server-core/jvm/src/io/ktor/sessions/Sessions.kt

```diff
@@ -224,7 +224,7 @@ private suspend fun <S : Any> SessionProviderData<S>.sendSessionData(call: Appli
 
 private data class SessionProviderData<S : Any>(var value: S?, val incoming: Boolean, val provider: SessionProvider<S>)
 
-private val SessionKey = AttributeKey<SessionData>("SessionKey")
```

Comment:
`SessionData` class is private so there is no reason to make it's key public. Also, it breaks compilation with the following error:

> 'public' property exposes its 'private' type argument SessionData

Reply:
Yes SessionData is private, making the key public won't help unless SessionData is also public and I am not sure if that is the correct thing to do.  I reverted and forgetting about this change unless you have have a better idea.